extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -605,6 +636,7 @@ export class AmpConsent extends AMP.BaseElement {
         return false;
       }
       // Prompt
+      this.shouldServerSync_ = true;
```

Comment:
i don't think this is the right place to set the bit. we need to wait until user responded to the UI.

the bit should also be `this.consentStateChangedViaPromptUI_`. once set true, remote sync should not override the current client cache.

Reply:
SG

Reply:
SG